django-backend/fecfiler/contacts/views.py

```diff
@@ -58,6 +62,72 @@ class ContactViewSet(CommitteeOwnedViewSet):
     ]
     ordering = ["-created"]
 
+    @action(detail=False)
+    def candidate_lookup(self, request):
+        q = request.GET.get("q")
+        if q is None:
+            return HttpResponseBadRequest()
+
+        max_fec_results = self.get_int_param_value(
+            request, "max_fec_results", default_max_fec_results, max_allowed_results
+        )
+
+        max_fecfile_results = self.get_int_param_value(
+            request,
+            "max_fecfile_results",
+            default_max_fecfile_results,
+            max_allowed_results,
+        )
+
+        query_params = urlencode({"q": q, "api_key": FEC_API_KEY})
+        url = "{url}?{query_params}".format(
```

Comment:
no need to homebrew this.  requests has got you covered https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls
```
params = urlencode({"q": q, "api_key": FEC_API_KEY}
requests.get(FEC_API_CANDIDATE_LOOKUP_ENDPOINT, params=params).json()
```

Reply:
https://github.com/fecgov/fecfile-web-api/pull/357

Reply:
Resolved with patch